src/main/sensors/voltage.c

```diff
@@ -191,7 +191,7 @@ void voltageMeterADCInit(void)
         voltageMeterADCState_t *state = &voltageMeterADCStates[i];
         memset(state, 0, sizeof(voltageMeterADCState_t));
 
-        biquadFilterInitLPF(&state->filter, VBAT_LPF_FREQ, 50000);
+        biquadFilterInitLPF(&state->filter, VBAT_LPF_FREQ, VBAT_LPF_FREQ, 20000); //50HZ Update
```

Comment:
Something is wrong here, this function doesn't take 4 arguments.

Reply:
ouch soor, will fix it :)

Reply:
Yes we should improve this with macro HZ_TO_INTERVAL_US(x) (1000000 / (x)) . I  think it should be a part of more complex PR because related magic conversions should be here and there.